File: iFlat/src/main/java/com/iflat/book/bean/Book.java
package com.iflat.book.bean;

/**
 * Created by tyriv on 2016/1/6.
 */
public class Book {

    private String id;
    private String category;
    private String name;
    private String author;
    private String publisher;
    private String owner;
    private int number;

    public int getNumber() {
        return number;
    }

    public void setNumber(int number) {
        this.number = number;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getPublisher() {
        return publisher;
    }

    public void setPublisher(String publisher) {
        this.publisher = publisher;
    }

    public String getOwner() {
        return owner;
    }

    public void setOwner(String owner) {
        this.owner = owner;
    }
}

File: iFlat/web/app/model/book/Book.js
Ext.define('iFlat.model.book.Book', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'book.id', mapping: 'id', type: 'string'},
        {name: 'book.category', mapping: 'category', type: 'string'},
        {name: 'book.name', mapping: 'name', type: 'string'},
        {name: 'book.author', mapping: 'author', type: 'string'},
        {name: 'book.publisher', mapping: 'publisher', type: 'string'},
        {name: 'book.owner', mapping: 'owner', type: 'string'},
        {name: 'book.number', mapping: 'number', type: 'number'}
    ]
});